pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -81,6 +81,7 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 			SupplementalGroups: kapi.SupplementalGroupsStrategyOptions{
 				Type: kapi.SupplementalGroupsStrategyRunAsAny,
 			},
+			SeccompProfiles: []string{"*"},
```

Comment:
Why are we not making this restrictive?  Do we have a value that is actually secure that we can use here?